libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ProbabilityDistributionChart.tsx

```diff
@@ -0,0 +1,244 @@
+      (_cohort, index) => {
+        return this.props.selectedFeatureBasedCohorts.includes(index);
+      }
+    );
+    const selectedCohorts = selectedDatasetCohorts.concat(
+      selectedFeatureBasedCohorts
+    );
+    const selectedCohortNames = selectedCohorts.map(
+      (cohort) => cohort.cohort.name
+    );
+
+    const probabilityOptions = this.getProbabilityOptions();
+
+    if (probabilityOptions.length === 0) {
+      return React.Fragment;
+    }
+
+    if (this.state.probabilityOption === undefined) {
+      this.setState({ probabilityOption: probabilityOptions[0] });
+      return React.Fragment;
+    }
+
+    const boxPlotData = selectedCohorts.map((cohort, index) => {
+      return calculateBoxPlotDataFromErrorCohort(
+        cohort,
+        index,
+        this.state.probabilityOption!.key.toString()
+      );
+    });
+    const outlierData = boxPlotData
+      .map((cohortBoxPlotData) => cohortBoxPlotData.outliers)
+      .map((outlierProbs, cohortIndex) => {
+        return outlierProbs.map((prob) => [cohortIndex, prob]);
+      })
+      .reduce((list1, list2) => list1.concat(list2), []);
+
+    const noCohortSelected =
+      this.props.selectedDatasetCohorts.length +
+        this.props.selectedFeatureBasedCohorts.length ===
+      0;
+
+    return (
+      <>
+        <Stack horizontal>
+          {!noCohortSelected && (
+            <Stack.Item className={classNames.verticalAxis}>
+              <DefaultButton
+                className={classNames.rotatedVerticalBox}
+                text={
+                  localization.ModelAssessment.ModelOverview
+                    .cohortSelectionButton
+                }
+                onClick={this.props.onChooseCohorts}
+              />
+            </Stack.Item>
+          )}
+          <Stack.Item className={classNames.chart}>
+            {noCohortSelected && (
+              <div className={classNames.placeholderText}>
+                <Text>
+                  {
+                    localization.ModelAssessment.ModelOverview
+                      .boxPlotPlaceholder
+                  }
+                </Text>
+              </div>
+            )}
+            {!noCohortSelected && (
+              <Stack>
+                <BasicHighChart
+                  id={"ProbabilityDistributionChart"}
+                  theme={theme}
+                  configOverride={{
+                    chart: {
+                      height: selectedCohorts.length * 40 + 120,
+                      inverted: true,
+                      type: "boxplot"
+                    },
+                    plotOptions: {
+                      bar: {
+                        dataLabels: {
+                          enabled: true
+                        }
+                      }
+                    },
+                    series: [
+                      {
+                        data: boxPlotData,
+                        fillColor: "#b2d6f2",
```

Comment:
hmm, can we replace the color with the fluentui constants, maybe cyanblue10 is close: https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/personas

or themeTertiary:
https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/theme-slots

Reply:
![image](https://user-images.githubusercontent.com/24683184/163586280-d73dd0d1-1e1d-4a00-bd57-8961f0565a10.png)

Reply:
We should stay with sematic colors for all non-chart related stuff
https://msdata.visualstudio.com/Vienna/_wiki/wikis/Workspace%20Portal%20Wiki/4919/Theming

For charts I remember there was a predefined color palette somewhere in our code.

Reply:
I'll check those, but this is the color that was given to me by Jingya. @minthigpen wdyt?

Reply:
It is indeed the same color as cyanBlue10 from SharedColors of fluentui, but I'm getting errors using both fluentui and office-ui-fabric-react in the same file. I'll leave it as is until we switch to fluentui for everything. There are many such occurrences in the code that we should clean up at that point.